build.yaml

```diff
@@ -2244,9 +2244,11 @@ steps:
      export HAIL_CURL_IMAGE={{ curl_image.image }}
      export HAIL_DEFAULT_NAMESPACE={{ default_ns.name }}
      export HAIL_NETCAT_UBUNTU_IMAGE={{ netcat_ubuntu_image.image }}
+     export HAIL_HAIL_BASE_IMAGE={{ hail_base_image.image }}
      export DOCKER_ROOT_IMAGE="gcr.io/{{ global.project }}/ubuntu:18.04"
      export HAIL_TEST_TOKEN_FILE=/user-tokens/tokens.json
      export HAIL_TEST_DEV_TOKEN_FILE=/dev-tokens/tokens.json
+     export HAIL_TOKEN="{{ token }}"
```

Comment:
Should we make sure this token shouldn't be used in other places in the tests? I think I just generated random tokens inside the tests.

Reply:
I modified the billing project creation. The other ones don't modify global state so it doesn't seem important.